### PR TITLE
Issue image - Allowing for user to choose image

### PR DESCRIPTION
**Pull request by Nanoemc**
Upgrade to htcdaskgateway that allows the user to specify the image for the client and scheduler to use.

### CODE REVIEW

**Comment by mapsacosta** [comment]
Hi @Nanoemc there are a few pycache files left in folder __pycache__ can you pls remove that folder? I think your commit also accidentally pushed a notebook called Untitled.ipynb so we need to get rid of that as well :)

Thanks!!

**Comment by mapsacosta** [comment]
Looks good to me, let me test a few corner cases and merge if successful

**Comment by mapsacosta** [comment]
Hi @Nanoemc any news on the new changes?

**Comment by Nanoemc** [comment]
@mapsacosta Sorry! I haven't had a chance to update it yet!

File: src/htcdaskgateway/cluster.py [hunk 1]
```diff
@@ -33,7 +44,7 @@ async def _stop_async(self):
 
         self.status = "closed"
     
-    def scale(self, n, **kwargs):
+    def scale(self, n):
```
Comment (mapsacosta) [inline src/htcdaskgateway/cluster.py:47]: @Nanoemc why were the **kwargs removed from this method?

**Comment by Nanoemc** [comment]
@mapsacosta Code is updated and I did some testing with the agc calver 2024 notebook, I was able to produce the plots.

File: src/htcdaskgateway/cluster.py (hunk 1 above)
Comment (Nanoemc) [reply src/htcdaskgateway/cluster.py:47]: I realized it didn't do anything, you would have to pass in kwargs when the user did cluster.scale(10). I thought you needed it to access the kwargs in gateway.new_cluster(), but that doesn't seem to work, I was just accessing stuff already stored in self. I don't think a user would need kwargs for scale, but maybe in the future?

Comment (mapsacosta) [reply src/htcdaskgateway/cluster.py:47]: It's in the [original code](https://github.com/dask/dask-gateway/blob/bf04d65def1c6d6ca758d14b849735e2368789c8/dask-gateway/dask_gateway/client.py#L1091) so if it doesn't make any difference we should keep it as it was before (with **kwargs)

Comment (Nanoemc) [reply src/htcdaskgateway/cluster.py:47]: Ah ok, I thought I had added it so I will add it back

File: src/htcdaskgateway/cluster.py [hunk 2]
```diff
@@ -54,7 +65,7 @@ def scale(self, n, **kwargs):
                 self.batchWorkerJobs.append(self.scale_batch_workers(n))
                 logger.debug(" New Cluster state ")
                 logger.debug(self.batchWorkerJobs)
-                return self.gateway.scale_cluster(self.name, n, **kwargs)
+                return self.gateway.scale_cluster(self.name, n)
```
Comment (mapsacosta) [inline src/htcdaskgateway/cluster.py:68]: oop we also need to add it back here

Comment (Nanoemc) [reply src/htcdaskgateway/cluster.py:68]: Oh shoot! I am on it!

Comment (mapsacosta) [reply src/htcdaskgateway/cluster.py:68]: Thank you :D
